muspinsim/muspinsim.xml

```diff
@@ -0,0 +1,77 @@
+<tool id="muspinsim" name="Simulate muon experiment spin dynamics (muspinsim)" version="0.1.0" python_template_version="3.5">
+    <requirements>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        pip install git+https://github.com/muon-spectroscopy-computational-project/muspinsim.git &&
+        ## remove spaces in names
+        mu_sim_name=\$(sed 's/ //g' <<< "$mu_sim.name") &&
+        ## create links
+        ln -s $mu_sim \$mu_sim_name &&
+        ## grab 'name' field from input file, if it doesn't exist, set name as 'muspinsim'
+        mu_out=\$(grep -A1 '^name' \$mu_sim_name | grep -v 'name' | sed -e 's/^[ \t]*//') &&
+        mu_out=\${mu_out:-"muspinsim"}.dat &&
+        log_out=\$(echo \$mu_sim_name | cut -d '.' -f1).log &&
+        ## if fitting data required, check if fitting data exists
+        ## create symlink and change filepath in input file to point to symlink
+        if grep '^fitting_data' $mu_sim; then
+            if [[ $mu_exp_in.name != 'None' ]]; then
```

Comment:
Using names with spaces, I got the error below. Slightly hard to tell, but I think wrapping `$mu_exp_in.name ` in quotes here will fix it.
```
/tmp/tmphrgcz7ls/job_working_directory/000/6/tool_script.sh: line 9: conditional binary operator expected
/tmp/tmphrgcz7ls/job_working_directory/000/6/tool_script.sh: line 9: syntax error near `data.dat'
/tmp/tmphrgcz7ls/job_working_directory/000/6/tool_script.sh: line 9: `pip install git+https://github.com/muon-spectroscopy-computational-project/muspinsim.git && mu_sim_name=$(sed 's/ //g' <<< "fitting data.in") && ln -s /tmp/tmphrgcz7ls/files/000/dataset_1.dat $mu_sim_name && mu_out=$(grep -A1 '^name' $mu_sim_name | grep -v 'name' | sed -e 's/^[ \t]*//') && mu_out=${mu_out:-"muspinsim"}.dat && log_out=$(echo $mu_sim_name | cut -d '.' -f1).log && if grep '^fitting_data' /tmp/tmphrgcz7ls/files/000/dataset_1.dat; then if [[ fitting data.dat != 'None' ]]; then mu_exp_in_name=$(sed 's/ //g' <<< "fitting data.dat") && ln -s /tmp/tmphrgcz7ls/files/000/dataset_2.dat $mu_exp_in_name && sed -iE "/^fitting_data/{n;s/([^)]*)/(\"$mu_exp_in_name\")/g}" $mu_sim_name; else echo "fitting data required, but no experiment data given" && exit 64; fi; fi && cat $mu_sim_name && muspinsim $mu_sim_name && mv $mu_out mu_out.dat && mv $log_out log_out.log'
```

Reply:
Ah, yep sorry forgot to test that part